File: front/aircorp/aircorp/src/components/Flight.js
import PurchaseButton from "./PurchaseButton"

const Flight = (flight) => {
    return (
        <div class="flight">
            <div class="flight-cell">
                {flight.flight[1].id}
            </div>
            <div class="flight-cell">
                {flight.flight[1].seats}
            </div>
            <div class="flight-cell">
                {flight.flight[1].seatPrice}
            </div>
            <div class="flight-cell">
                {flight.flight[1].date}
            </div>
            <div class="flight-cell">
                <PurchaseButton id={flight.flight[1].id} />
            </div>
        </div>
    )
}

export default Flight

File: aircorp/src/main/java/com/aircorp/aircorp/Flight.java
package com.aircorp.aircorp;

import java.util.Date;

public class Flight {
    public Flight(String id, Integer seats, Integer seat_price, Date date) {
        this.id = id;
        this.seats = seats;
        this.seat_price = seat_price;
        this.date = date;
    }

    public String id;
    public Integer seats;
    public Integer seat_price;
    public Date date;
}
